linux_os/guide/system/permissions/restrictions/coredumps/coredump_disable_storage/rule.yml

```diff
@@ -19,6 +19,7 @@ identifiers:
 
 references:
     ospp: FMT_SMF_EXT.1
+    srg: SRG-OS-000480-GPOS-00227
```

Comment:
Should coredumps still remain disabled? Perhaps it's time to let this rule go?

Reply:
@shawndwells why? coredumps can expose some sensitive information.

Reply:
I agree with @redhatrises .  This rule should stay and protects against people that enable core-dumps in the installer.